recharges/tasks.py

```diff
@@ -167,4 +224,37 @@ def run(self, recharge_id, **kwargs):
         elif status == 4:
             return "airtime request for %s is unrecoverable" % recharge.msisdn
 
+
 hotsocket_get_airtime = Hotsocket_Get_Airtime()
+
+
+class Check_Hotsocket_Status(Task):
+
+    """
+    Task to check hotsocket recharge request and sets the recharge model
+    status to successful if the airtime has been loaded to the user's phone.
+    """
+    name = "recharges.tasks.Check_Hotsocket_Status"
+
+    def prep_hotsocket_status_dict(self, recharge_id, hs_reference):
+
+        hotsocket_data = {
+            'username': settings.HOTSOCKET_API_USERNAME,
+            'as_json': True,
+            'token': get_token(),
+            'reference': recharge_id + settings.HOTSOCKET_REFBASE,
+            'hotsocket_ref': hs_reference
```

Comment:
From the documentation:
`Status can handle your reference or the reference HOTSOCKET returned - just
leave the other reference blank (it’s XOR, so don’t send both, you will get an error message).`